src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -366,8 +366,7 @@ class AgoraBusiness(permissionsDataSource: PermissionsDataSource) extends LazyLo
       case (alias, owner, permissions) if permissions.hasPermission(AgoraPermissions.Manage) => (alias, owner)
     })
 
-    // In Scala 2.13 this can be replaced with a .groupMap()
-    val ownersAndAliasesMap = ownersAndAliases.groupBy(_._1).mapValues(_.map(_._2))
+    val ownersAndAliasesMap = ownersAndAliases.groupMap(_._1)(_._2)
```

Comment:
TYVM @kshakir

Reply:
Since this is a new function, I think type annotations on `ownersAndAliases` and `ownersAndAliasesMap` would make it easier to work out what's going on.